### PR TITLE
[SD-3857] - (imp) - Add a separate label for Embargoed items 

### PR DESCRIPTION
**Pull request by sjunaid**

### CODE REVIEW

**Comment by ioanpocol** [comment]
:+1:

File: scripts/superdesk/filters.js [hunk 1]
```diff
@@ -78,6 +78,34 @@
...
+                var COMPARE_FORMAT = 'YYYY-M-D';
+                var DISPLAY_DATE_FORMAT = 'D. MMMM YYYY [at] HH:mm';
+                var DISPLAY_CDATE_FORMAT = 'D. MMMM [at] HH:mm';
+                var DISPLAY_DAY_FORMAT = 'dddd, ';
+                var DISPLAY_TODAY_FORMAT = '[Today], ';
+                var rday, rdate;
+
+                var date = moment.utc(input);
+
+                date.local(); // switch to local time zone
```
Comment (akintolga) [inline scripts/superdesk/filters.js]: I don't think it should switch to local date. We need to show what ever the user sets the embargo date/time without conversion.

**Comment by sjunaid** [comment]
minimized the scope of this improvement and removed the embargoTS title for time being.

**Comment by mdhaman** [comment]
:+1:

**Comment by petrjasek** [comment]
:+1:

**Pull request merged**